src/sentry.ts

```diff
@@ -3,22 +3,24 @@ import * as Sentry from '@sentry/react';
 
 import { history } from './browserhistory';
 
-Sentry.init({
-  dsn:
-    'https://4fc1290ffbf3e98712374e16e8cea4ac@o4504154328465408.ingest.us.sentry.io/4508889530236928',
-  integrations: [
-    Sentry.reactRouterV5BrowserTracingIntegration({ history }),
-    Sentry.replayIntegration(),
-  ],
-  tracePropagationTargets: ['localhost'],
-  // Session Replay
-  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
-  replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
+if (process.env.NODE_ENV !== 'development') {
```

Comment:
nit: prefer non-negated for clarity 

```suggestion
if (process.env.NODE_ENV === 'production') {
```

Reply:
say less